extension/docs/HowToRun.md

```diff
@@ -73,8 +73,8 @@ Multiple child attributes can be provided in the `adyen` attribute. Each direct
 | Name               | Content                                                                                                                                                  | Required | Default value (only for test environment)            |
 | ------------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------- | -------- | ---------------------------------------------------- |
 | `apiKey`           | You'll be making API requests that are authenticated with an [API key](https://docs.adyen.com/user-management/how-to-get-the-api-key#page-introduction). | YES      |                                                      |
-| `apiBaseUrl`       | [Checkout endpoint](https://docs.adyen.com/development-resources/live-endpoints#checkout-endpoints) of Adyen.                                            | NO       | `https://checkout-test.adyen.com/v52`                |
-| `legacyApiBaseUrl` | [Standart payment endpoint](https://docs.adyen.com/development-resources/live-endpoints#standard-payments-endpoints) of Adyen.                           | NO       | `https://pal-test.adyen.com/pal/servlet/Payment/v52` |
+| `apiBaseUrl`       | [Checkout endpoint](https://docs.adyen.com/development-resources/live-endpoints#checkout-endpoints) of Adyen.                                            | NO       | `https://checkout-test.adyen.com/v52` (even though it is not required, you **need** to spcefiy a URL for live environment)             |
```

Comment:
@ahmetoz @leungkinghin @butenkor WDYT about this change? We had an issue that it is not clear enough that for production it needs another value. See: https://github.com/commercetools/commercetools-adyen-integration/pull/805.
Right now we have only in the column name written `Default value (only for test environment)`. Do you think this change is good enough or sth more needs to be done here?

Reply:
The PR is a bit confusing, I guess you are mixing different fork PRs. 
I think it's enough with the new documentation update.

Reply:
Yes, just check this part only. The other changes are not related.